src/OSSupport/Network.h

```diff
@@ -0,0 +1,215 @@
+
+// Network.h
+
+// Declares the classes used for the Network API
+
+
+
+
+
+#pragma once
+
+
+
+
+
+/** Interface that provides the methods available on a single TCP connection. */
+class cTCPLink
```

Comment:
Why put an interface definition that has some 30 lines into a separate file, just for the sake of it?

`cCallbacksPtr` is shared between multiple links that are created by the `cServerHandle` for incoming connections.